Watchman.DomainModel/Settings/IMappedConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Watchman.DomainModel.Settings
+{
+    public interface IMappedConfiguration
+    {
+        ulong ServerId { get; set; }
```

Comment:
lepiej private set
lepiej ServerId nie zmieniać podczas życia obiektu 

jak już to tworzyć nowy obiekt w oparciu o poprzedni